bench/irmin-pack/trace_replay.ml

```diff
@@ -433,6 +432,7 @@ module Make (Store : Store) = struct
               (Hashtbl.find t.key_per_commit_idx i
               |> Store.Backend.Commit.Key.to_hash)
           in
+          t.current_commit_idx <- i;
```

Comment:
It seems strange to set `current_commit_idx` to `i` here. I think it should either be renamed to `prev_commit_idx` or be updated to `(i + 1)`

Reply:
I wasn't quite sure the best name for this new variable since it depends on perspective for whether it is "current" or "previous". Prior to this PR, we used `i - 1` to represent the head commit id -- ie the last commit to be replayed -- which I wanted to remove. Setting this variable _after_ the `i` commit is replayed allows us to just use `i` instead of `i - 1`. What about `latest_commit_idx` or something like that?